.github/workflows/lint-testdata.yml

```diff
@@ -0,0 +1,85 @@
+name: Testdata Lint Check
+
+# Trigger the workflow on pull requests and direct pushes to any branch
+on:
+  push:
+  pull_request:
+
+jobs:
+  check-lint-for-project-v2:
+    uses: kubernetes-sigs/kubebuilder/.github/workflows/lint-testdata.yml@master
```

Comment:
I guess we will be using `lint-testdata-template.yml` here?

Reply:
Thx for catching! 👍